contacts/forms.py

```diff
@@ -14,4 +15,20 @@ class Meta:
             'zip_code',
             'phone_number',
             'email',
+            'birthday',
         ]
+
+        model = Note
```

Comment:
The approach took    here makes sense, but the assignment doesn't require it. The `Note` model gets its own form that will be rendered at the url that you made to view an individual contact.